staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -178,6 +180,10 @@ func (c *webhookConverter) Convert(in runtime.Object, toGV schema.GroupVersion)
 		out.SetAPIVersion(toGV.String())
 		return out, nil
 	}
+
+	trace := utiltrace.New(fmt.Sprintf("Conversion webhook request: %s", c.name))
```

Comment:
this is not specific to this PR

`c.name` doesn't uniquely identify one `webhookConverter`. We create identical converters with the same name for the storage of each CR version:
https://github.com/kubernetes/kubernetes/blob/59abf0fe3ecf88a0e5019d9f4c9067a6b46d63a4/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go#L517-L518

(perhaps it's okay to trace the performance for these converters together, since we expect them to work identically)

Reply:
I'd like to also know the following when a conversion is taking longer than expected:

- is this converting a single CR or a list?
- what's the target GV?
- (what's the input GV?)

Reply:
+1, I'll add these and review all the traces for similar relevant bits of information that we should include in the traces